pkg/abi/nvgpu/classes.go

```diff
@@ -333,6 +334,37 @@ type NV00F8_ALLOCATION_PARAMETERS struct {
 	Map        nv00f8Map
 }
 
+// From src/common/sdk/nvidia/inc/class/cl00e0.h
+const (
+	NV_MEM_EXPORT_UUID_LEN = 16
+)
+
+// NV_EXPORT_MEM_PACKET is from
+// src/common/sdk/nvidia/inc/class/cl00e0.h
+//
+// +marshal
+type NV_EXPORT_MEM_PACKET struct {
+	UUID   [NV_MEM_EXPORT_UUID_LEN]uint8
+	Opaque [16]uint8
+}
+
+// NV00FD_ALLOCATION_PARAMETERS is the alloc param type for NV_MEMORY_MULTICAST_FABRIC
+// from src/common/sdk/nvidia/inc/class/cl00fd.h
+//
+// +marshal
+type NV00FD_ALLOCATION_PARAMETERS struct {
+	ExpPacket  NV_EXPORT_MEM_PACKET
+	Index      uint16
+	_          [6]byte
+	Alignment  uint64
+	AllocSize  uint64
+	PageSize   uint32
+	AllocFlags uint32
+	NumGPUs    uint32
+	_          uint32
+	POsEvent   P64
+}
```

Comment:
This struct has changed 2 times in the driver version window we support: https://github.com/NVIDIA/open-gpu-kernel-modules/blame/caa2dd11a0db42e5d6130f627dd15ef404b7597d/src/common/sdk/nvidia/inc/class/cl00fd.h#L87

Once in `550.40.07`, but this change was just cosmetic. Two newlines were added in the struct definition.
And once in `545.23.06`, this was significant.  The first two fields (expPacket and index) were added. We will need to add versioning support for `545.23.06`.